wherehows-web/app/services/notifications.ts

```diff
@@ -152,7 +153,7 @@ const notificationHandlers: INotificationHandler = {
     };
     // Set default values for button text if none are provided by consumer
     props = { dismissButtonText: 'No', confirmButtonText: 'Yes', ...props };
-    const { dismissButtonText, confirmButtonText } = props;
+    const { dismissButtonText, confirmButtonText, onDialogToggle } = props;
```

Comment:
Maybe it's the react in me speaking, but I thought `props` was like passed in component properties. hahaha

Reply:
haha, react? what's that?, jk not in this case :)